### PR TITLE
Adding issue to update constant params

### PR DESCRIPTION
**Pull request by mariolo1985**
This issue is to address the constants defined in the test file `src/ride-hailing/MissionParams.test.ts`. We would like help updating this test file to allow updating the parameters per unit test.

### CODE REVIEW

File: issues/davjs/MissionParms/01-mission-params.md [hunk 1]
```diff
@@ -20,22 +20,41 @@ This repo contains the DAV JavaScript SDK. This SDK allows developers to build a
...
+    protocol: 'ride_hailing',
+    type: 'mission',
+    price: undefined,
+    vehicleId: undefined,
+};
+```
 
+These parameters will need to accommodate additional unit tests. 
 
+Please update these parameters to be more dynamic by defining them to be mutable similar to `src/vessel-charging/MissionParams.test.ts`.
```
Comment (TalAter) [inline issues/davjs/MissionParms/01-mission-params.md:45]: `ride-hailing` not `vessel-charging`

**Review by TalAter — CHANGES_REQUESTED** [review]
Looks good. Please fix the small comment I made and post it as a new issue to the repo. Let me know once you do and I will add the appropriate labels
👍

File: issues/davjs/MissionParms/01-mission-params.md (hunk 1 above)
Comment (mariolo1985) [reply issues/davjs/MissionParms/01-mission-params.md:45]: What helped me understand this issue was the example you pointed me to at `src/vessel-charging/MissionParams.test.ts`. My thought is to provide the same example to a collaborator for additional context of what the task is asking for.

After re-reading this line. I would like to update it to:
> Please update these parameters to be dynamic and mutable. A similar example can be found in `src/vessel-charging/MissionParams.test.ts`.

What do you think?

Comment (TalAter) [reply issues/davjs/MissionParms/01-mission-params.md:45]: Right. My mistake.
Looks good.

**Review by TalAter — APPROVED** [review]

**Pull request merged**